Make build fails with error

**Describe the bug**
`make -j1 build` fails with error

**To Reproduce**
Steps to reproduce the behavior:
1. Download https://github.com/lensapp/lens/archive/v4.1.1.tar.gz
2. Decompress and enter intro directory
3. Do `make -j1 build`
4. After several time, see error
```
[compile:renderer] yarn run compile:renderer exited with code 0
  • electron-builder  version=22.7.0 os=5.10.16-arch1-1
  • loaded configuration  file=package.json ("build" field)
  • rebuilding native dependencies  dependencies=node-pty@0.9.0 platform=linux arch=x64
  • rebuilding native dependency  name=node-pty version=0.9.0
  • packaging       platform=linux arch=x64 electron=9.4.0 appOutDir=dist/linux-unpacked
  • downloading     url=https://github.com/electron/electron/releases/download/v9.4.0/electron-v9.4.0-linux-x64.zip size=73 MB parts=8
  • downloaded      url=https://github.com/electron/electron/releases/download/v9.4.0/electron-v9.4.0-linux-x64.zip duration=5.161s
  • file source doesn't exist  from=/build/lens/src/lens-4.1.1/locales
  • building        target=snap arch=x64 file=dist/Lens-4.1.1.amd64.snap
  • building        target=AppImage arch=x64 file=dist/Lens-4.1.1.x86_64.AppImage
  • downloading     url=https://github.com/electron-userland/electron-builder-binaries/releases/download/appimage-12.0.1/appimage-12.0.1.7z size=1.6 MB parts=1
  • downloading     url=https://github.com/electron-userland/electron-builder-binaries/releases/download/snap-template-4.0-2/snap-template-electron-4.0-2-amd64.tar.7z size=1.5 MB parts=1
  • downloaded      url=https://github.com/electron-userland/electron-builder-binaries/releases/download/snap-template-4.0-2/snap-template-electron-4.0-2-amd64.tar.7z duration=646ms
  • downloaded      url=https://github.com/electron-userland/electron-builder-binaries/releases/download/appimage-12.0.1/appimage-12.0.1.7z duration=703ms
  • downloading     url=https://github.com/electron-userland/electron-builder-binaries/releases/download/appimage-12.0.1/appimage-12.0.1.7z size=1.6 MB parts=1
  • downloaded      url=https://github.com/electron-userland/electron-builder-binaries/releases/download/appimage-12.0.1/appimage-12.0.1.7z duration=705ms
  • cannot move downloaded into final location (another process downloaded faster?)  path=/build/.cache/electron-builder/appimage/appimage-12.0.1
                                                                                     tempFile=/build/.cache/electron-builder/appimage/680898106
                                                                                     error=rename /build/.cache/electron-builder/appimage/680898106 /build/.cache/electron-builder/appimage/appimage-12.0.1: file exists
  • building        target=deb arch=x64 file=dist/Lens-4.1.1.amd64.deb
  ⨯ Please specify project homepage, see https://electron.build/configuration/configuration#Metadata-homepage  stackTrace=
                                                                                                                 Error: Please specify project homepage, see https://electron.build/configuration/configuration#Metadata-homepage
                                                                                                                     at FpmTarget.computeFpmMetaInfoOptions (/build/lens/src/lens-4.1.1/node_modules/app-builder-lib/src/targets/fpm.ts:82:13)
                                                                                                                     at FpmTarget.build (/build/lens/src/lens-4.1.1/node_modules/app-builder-lib/src/targets/fpm.ts:135:24)
                                                                                                                     at /build/lens/src/lens-4.1.1/node_modules/app-builder-lib/src/platformPackager.ts:133:11
                                                                                                                     at async Promise.all (index 0)
                                                                                                                     at AsyncTaskManager.awaitTasks (/build/lens/src/lens-4.1.1/node_modules/builder-util/src/asyncTaskManager.ts:65:25)
                                                                                                                     at Packager.doBuild (/build/lens/src/lens-4.1.1/node_modules/app-builder-lib/src/packager.ts:456:5)
                                                                                                                     at executeFinally (/build/lens/src/lens-4.1.1/node_modules/builder-util/src/promise.ts:12:14)
                                                                                                                     at Packager._build (/build/lens/src/lens-4.1.1/node_modules/app-builder-lib/src/packager.ts:373:31)
                                                                                                                     at Packager.build (/build/lens/src/lens-4.1.1/node_modules/app-builder-lib/src/packager.ts:337:12)
                                                                                                                     at executeFinally (/build/lens/src/lens-4.1.1/node_modules/builder-util/src/promise.ts:12:14)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
make: *** [Makefile:63: build] Error 1
```

**Environment (please complete the following information):**
- Lens Version: 4.1.1
- OS: ArchLinux
- Installation method: Trying to build from source
- Versions: nodejs-lts-erbium-12.20.1-1  npm-6.14.11-1  yarn-1.22.10-1

--- a/package.json
+++ b/package.json
@@ -2,6 +2,7 @@
   "name": "open-lens",
   "productName": "OpenLens",
   "description": "OpenLens - Open Source IDE for Kubernetes",
+  "homepage": "https://github.com/lensapp/lens",
   "version": "5.0.0-beta.6",
   "main": "static/build/main.js",
   "copyright": "© 2021 OpenLens Authors",

